src/app/themes/theme.js

```diff
@@ -68,32 +74,40 @@ const theme = createMuiTheme({
       fontSize: '0.875em',
     },
   },
+/*End of Typography*/
+
+/*Start of Overides */
   overrides: {
-    MuiPaper: {
-      rounded: {
-        borderRadius: '8px',
+    MuiAccordionSummary: {
+      root: {
+        backgroundColor: '#f5f9fd',
```

Comment:
Are we getting rid of the default slight grey color that's on the accordion headers? That seems like a big change to make in this PR

Reply:
I can take this out of this PR and discuss the topic next week